### PR TITLE
CSP-1131 Added attendance section in member profile page

### PR DESCRIPTION
**Pull request by ManishAgarwalDfE**

### CODE REVIEW

**Review by DevenKShah — APPROVED** [review]

File: src/SFA.DAS.Admin.Aan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs [hunk 1]
```diff
@@ -107,4 +139,92 @@ public void Get_ShouldReturnUpcomingEventsInEventDateOrder()
...
+    {
+        // Arrange 
+        await Init();
+
+        // Act
+        var viewResult = result as ViewResult;
+        var viewModel = viewResult!.Model as AmbassadorProfileViewModel;
+
+        // Assert
+        Assert.Multiple(() =>
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.Admin.Aan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs:180]: This assert multiple is unnecessary

File: src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml [hunk 2]
```diff
@@ -3,7 +3,26 @@
 @model MemberActivitiesViewModel;
 
 <div>
-    <h3 class="govuk-heading-m">Upcoming events (@Model.FutureEventsCount)</h3>
+    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-2">@Model.FirstName's activities</h1>
+    <p>Last event sign up: @Model.LastEventSignUpDate</p>
+    <h3 class="govuk-heading-m">
+        In the past 12 months
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml:9]: Ticket content reads "In the last 12 months"

EDIT Looks like the ticket is inconsistent between "last 12" and "past 12", worth double checking

File: src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml [hunk 3]
```diff
@@ -3,7 +3,26 @@
...
+    </h3>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half">
+            <div class="app-figure">
+                <p class="govuk-body app-figure__figure">@Model.EventsAttendedCount</p>
+                <p class="govuk-body app-figure__label">Events attended</p>
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml:15]: "Events attended" or "attended events" content is inconsistent on the ticket, worth double checking

File: src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml [hunk 4]
```diff
@@ -3,7 +3,26 @@
...
+            </div>
+        </div>
+        <div class="govuk-grid-column-one-half">
+            <div class="app-figure">
+                <p class="govuk-body app-figure__figure">@Model.SchoolEventsAttendedCount</p>
+                <p class="govuk-body app-figure__label">School events attended</p>
```
Comment (PeterDavidWilliams) [inline src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml:21]: "School events attended" or "attended school events" content is inconsistent on the ticket, worth double checking

File: src/SFA.DAS.Admin.Aan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs (hunk 1 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Admin.Aan.Web.UnitTests/Controllers/MemberProfileControllerTests.cs]: Done

File: src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml (hunk 2 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml:9]: We normally followed figma when there is contradiction between the ticket and figma.

File: src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml (hunk 3 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml:15]: We normally follow figma when there is contradiction between the ticket and figma.

File: src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml (hunk 4 above)
Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml:21]: We normally follow figma when there is contradiction between the ticket and figma.

**Review by PeterDavidWilliams — APPROVED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**